app/src/org/commcare/android/database/DbUtil.java

```diff
@@ -35,6 +35,11 @@ public static void setDBUtilsPrototypeFactory(PrototypeFactory factory) {
     }
 
 
+    /**
+     * Basically this is our PrototypeManager for Android
+     * @param c
+     * @return
```

Comment:
Please remove javadoc parameter entries that don't have descriptions (`@return` & `@param c`)